README-pt.md

```diff
@@ -1,76 +1,137 @@
 
-Depois que seu **pull request** for criado, um revisor do Kubernetes assumirá a responsabilidade de fornecer um feedback claro e objetivo. Como proprietário do pull request, **é sua responsabilidade modificar seu pull request para abordar o feedback que foi fornecido a você pelo revisor do Kubernetes.** Observe também que você pode acabar tendo mais de um revisor do Kubernetes para fornecer seu feedback ou você pode acabar obtendo feedback de um revisor do Kubernetes que é diferente daquele originalmente designado para lhe fornecer feedback. Além disso, em alguns casos, um de seus revisores pode solicitar uma revisão técnica de um [revisor de tecnologia Kubernetes](https://github.com/kubernetes/website/wiki/Tech-reviewers) quando necessário. Os revisores farão o melhor para fornecer feedback em tempo hábil, mas o tempo de resposta pode variar de acordo com as circunstâncias.
+## Pré-requisitos
 
-Para mais informações sobre como contribuir com a documentação do Kubernetes, consulte:
-
-* [Comece a contribuir](https://kubernetes.io/docs/contribute/start/)
-* [Preparando suas alterações na documentação](http://kubernetes.io/docs/contribute/intermediate#view-your-changes-locally)
-* [Usando Modelos de Página](http://kubernetes.io/docs/contribute/style/page-templates/)
-* [Guia de Estilo da Documentação](http://kubernetes.io/docs/contribute/style/style-guide/)
-* [Localizando documentação do Kubernetes](https://kubernetes.io/docs/contribute/localization/)
+Para usar este repositório, você precisa instalar:
 
-Você pode contactar os mantenedores da localização em Português em:
+- [npm](https://www.npmjs.com/)
+- [Go](https://golang.org/)
+- [Hugo (versão Extended)](https://gohugo.io/)
+- Um container runtime, por exemplo [Docker](https://www.docker.com/).
 
-* Felipe ([GitHub - @femrtnz](https://github.com/femrtnz))
-* [Slack channel](https://kubernetes.slack.com/messages/kubernetes-docs-pt)
+Antes de você iniciar, instale as dependências, clone o repositório e navegue até o diretório:
 
-## Executando o site localmente usando o Docker
+```
+git clone https://github.com/kubernetes/website.git
+cd website
+```
 
-A maneira recomendada de executar o site do Kubernetes localmente é executar uma imagem especializada do [Docker](https://docker.com) que inclui o gerador de site estático [Hugo](https://gohugo.io).
+O website do Kubernetes utiliza o [tema Docsy Hugo](https://github.com/google/docsy#readme). Mesmo se você planeje executar o website em um container, é altamente recomendado baixar os submódulos e outras dependências executando o seguinte comando:
 
-> Se você está rodando no Windows, você precisará de mais algumas ferramentas que você pode instalar com o [Chocolatey](https://chocolatey.org). `choco install make`
+```
+# pull in the Docsy submodule
```

Comment:
If you want, you can also translate this comment.

Reply:
Comment translated.